.rubocop.yml

```diff
@@ -17,6 +17,9 @@ Metrics/ClassLength:
 Metrics/BlockLength:
   ExcludedMethods: ['describe']
   Max: 30
+Metrics/ModuleLength:
+  ExcludedMethods: ['describe']
+  Max: 200

```

Comment:
You should not overwrite the default config file provided by Microverse. Instead, you can use the inline rule disable approach to get rid of an error that you think going to take to many tasks to resolve it. But if you think that a ruleset is important and it should be added check out the instruction in this [link](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)